source/tutorials/metadata.rst

```diff
@@ -0,0 +1,141 @@
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv \
+     --m-input-file faith_pd_vector.qza \
+     --o-visualization tabulated-combined-metadata.qzv
+
+Merging is based on the identifiers specified in the first column of each metadata file (i.e. sample IDs or feature IDs). The resulting metadata after the merge will contain the intersection of the identifiers across all of the specified files. Merging does not modify any of the input files.
+
+.. question::
+   Modify the command above to merge the `evenness vector`_ of ``SampleData[AlphaDiversity]`` after the Faith's PD vector. What happens when merging the three artifacts? How many columns are present in the resulting metadata visualization? How many of those columns represent the sample IDs? How many of those columns represent ``SampleData[AlphaDiversity]`` metrics?
+
+Metadata merging is supported anywhere that metadata is accepted in QIIME 2. For example, it might be interesting to color an Emperor plot based on the study metadata, or the sample alpha diversities. This can be accomplished by providing both the sample metadata mapping file *and* the ``SampleData[AlphaDiversity]`` artifact:
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/core-metrics-results/unweighted_unifrac_pcoa_results.qza
+   :saveas: unweighted_unifrac_pcoa_results.qza
+
+.. command-block::
+   qiime emperor plot \
+     --i-pcoa unweighted_unifrac_pcoa_results.qza \
+     --m-metadata-file sample-metadata.tsv \
+     --m-metadata-file faith_pd_vector.qza \
+     --o-visualization unweighted-unifrac-emperor-with-alpha.qzv
+
+.. question::
+   What happens to the visualization if the order of the metadata files is reversed in the above ``emperor plot``? What about in the ``metadata tabulate`` example above?
+
+Exploring feature metadata
+--------------------------
+
+As noted above, metadata can exist for either axis of a ``FeatureTable``: the sample axis or the feature axis. The previous examples in this tutorial were focused on sample-oriented metadata.
+
+To get started with feature metadata, first download the example files:
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/rep-seqs.qza
+   :saveas: rep-seqs.qza
+
+.. download::
+   :url: https://docs.qiime2.org/2017.6/data/tutorials/moving-pictures/taxonomy.qza
+   :saveas: taxonomy.qza
+
+Next, merge the two metadata files, and ``tabulate`` them:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file rep-seqs.qza \
+     --m-input-file taxonomy.qza \
+     --o-visualization tabulated-feature-metadata.qzv
+
+The resulting table shows the joined metadata files with a column of the the feature IDs, a column of the representative sequences, a column of the taxonomic assignments, and lastly, a column of the assignment confidence.
+
+.. question::
+   Are all QZA files metadata? Are all metadata files QZAs?
```

Comment:
Change to:

*Are all .qza files viewable as metadata? Are all metadata files stored as .qza files?*

I don't think users have enough to work on to answer the first question. It's eluded to in a couple places but it's easy to miss. What about linking to an artifact (e.g. feature table) that isn't viewable as metadata to give users a nudge in the right direction?